docs/ambiguity.md

```diff
@@ -1,79 +1,363 @@
+  It also includes helper functions, e.g. to fetch the current time zone offset for a particular exact time.
+- The [`Temporal.LocalDateTime`](./localdatetime.md) type encapsulates all of the types above: an exact time (like a [`Temporal.Instant`](./instant.md)), its wall-clock equivalent (like a [`Temporal.DateTime`](./datetime.md)), and the time zone that links the two (like a [`Temporal.TimeZone`](./timezone.md)).
+
+There are two ways to get a human-readable calendar date and clock time from a `Temporal` type that stores exact time.
+
+- If the exact time is already represented by a [`Temporal.LocalDateTime`](./localdatetime.md) instance then the wall-clock time values are trivially available using the properties and methods of that type, e.g. [`.year`](./localdatetime.md#year), [`.hour`](./localdatetime.md#hour), [`.getFields()`](./localdatetime.md#getFields), or [`.toLocaleString()`](./localdatetime.md#toLocaleString).
+- However, if the exact time is represented by a [`Temporal.Instant`](./instant.md), use a time zone and optional calendar to create a [`Temporal.LocalDateTime`](./localdatetime.md). Example:
+
+<!-- prettier-ignore-start -->
+```javascript
+instant = Temporal.Instant.from('2019-09-03T08:34:05Z');
+formatOptions = {
+  era: 'short',
+  year: 'numeric',
+  month: 'short',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+  second: 'numeric'
+};
+
+ldt = instant.toLocalDateTime(`Asia/Tokyo`);
+  // => 2019-09-03T17:34:05+09:00[Asia/Tokyo]
+ldt.toLocaleString('en-us', { ...formatOptions, calendar: ldt.calendar });
+  // => "Sep 3, 2019 AD, 5:34:05 PM"
+ldt.year;
+  // => 2019
+ldt = instant.toLocalDateTime(`Asia/Tokyo`, 'iso8601').toLocaleString('ja-jp', formatOptions);
+  // => 2019-09-03T17:34:05+09:00[Asia/Tokyo]
+  // this is identical to the result of toLocalDateTime() above
+
+ldt = instant.toLocalDateTime(`Asia/Tokyo`, 'japanese');
+  // => 2019-09-03T17:34:05+09:00[Asia/Tokyo][c=japanese]
+ldt.toLocaleString('en-us', { ...formatOptions, calendar: ldt.calendar });
+  // => "Sep 3, 1 Reiwa, 5:34:05 PM"
+ldt.year;
+  // => 1
+```
+<!-- prettier-ignore-end -->
+
+Conversions from calendar date and/or wall clock time to exact time are also supported:
+
+<!-- prettier-ignore-start -->
+```javascript
+// Convert various local time types to an exact time type by providing a time zone
+date = Temporal.Date.from('2019-12-17');
+ldt = date.toLocalDateTime(`Asia/Tokyo`);
+  // => 2019-12-17T00:00+09:00[Asia/Tokyo]
+ldt = date.toLocalDateTime(`Asia/Tokyo`, Temporal.Time.from('10:00'));
+  // => 2019-12-17T10:00+09:00[Asia/Tokyo]
+time = Temporal.Time.from('14:35');
+ldt = time.toLocalDateTime(`Asia/Tokyo`, Temporal.Date.from('2020-08-27'));
+  // => 020-08-27T14:35+09:00[Asia/Tokyo]
+dateTime = Temporal.DateTime.from('2019-12-17T07:48');
+ldt = dateTime.toLocalDateTime(`Asia/Tokyo`);
+  // => 2019-12-17T07:48+09:00[Asia/Tokyo]
+
+// Get the exact time in seconds, milliseconds, or nanoseconds
```

Comment:
```suggestion
// Get the exact time in seconds, milliseconds, or nanoseconds since the UNIX epoch.
```

Reply:
Good point. I'll update the code to be milliseconds.